lib/Lift.cpp

```diff
@@ -569,6 +569,22 @@ CreateConstFromMemory(const uint64_t addr, llvm::Type *type,
       }
     } break;
 
+    case llvm::Type::StructTyID: {
```

Comment:
Could you add a comment about what this is supposed to do, at a high level? It looks like it will statically initialize structure members?

Reply:
It statically initializes the global variables of type struct. Currently, it only supports the initialization for integer, pointer, or array types. I will update the comments. 
There is a PR from Alessandro with the same changes and it should be updated after the merge.